doc/docs/Exploiting_Symmetry.md

```diff
@@ -41,11 +41,23 @@ To exploit $E_z$ or $H_z$ symmetry, you don't have to do anything special: if yo
 
 ### Cylindrical Symmetry
 
-If your structure has cylindrical symmetry (also known as the $C_{\infty\mathrm{v}}$ group), i.e. continuous rotational symmetry around the $z$ axis, then Meep can exploit this by running the simulation in cylindrical/polar coordinates.
+If your structure has cylindrical symmetry (also known as the $C_{\infty\mathrm{v}}$ group), i.e. continuous rotational symmetry around the $z$ axis, then Meep can exploit this by running the simulation in [cylindrical/polar coordinates](https://en.wikipedia.org/wiki/Cylindrical_coordinate_system).
 
 To do this, you must set `dimensions = mp.CYLINDRICAL` in the Python script, and then the vectors are interpreted as $(r,\phi,z)$ triplets instead of $(x,y,z)$.
 
-In a cylindrical simulation, fields can be written in the form of a function of $(r,z)$ multiplied by a function $\exp(im\phi)$ for the $\phi$ dependence, where $m$ is a user-specified parameter (related to the angular momentum of the field). If the cell includes the $r=0$ origin, then $m$ must be an integer to be single-valued at the origin. Otherwise, $m$ can be arbitrary (this is useful e.g. for bend simulations).
+In a cylindrical simulation, the fields can be written in the form of a function of $(r,z)$ multiplied by a function $\exp(im\phi)$ for the $\phi$ dependence, where $m$ is a user-specified parameter (related to the angular momentum of the field). For reference, Maxwell's equations in cylindrical coordinates and a schematic of the Yee grid are shown below.
+
+![](images/Maxwell_equations_cylindrical.png#center)
+
+![](images/Yee_grid_cylindrical.png#center)
+
+The fields at $r = 0$ must obey certain symmetries determined by the value of $m$:
+
+* $m = 0$: fields are rotation-invariant. At $r = 0$, this means that $E_z$ and $H_z$ can be nonzero, but all the other field components must be zero at the origin (since they are otherwise not rotation-invariant).
+* $m = \pm 1$: fields flip sign when rotated by π radians. At $r = 0$, this means that $E_\phi, E_r, H_z, H_r$ can be nonzero, but the $E_z, H_z$ fields must be zero.
+* $|m| > 1$: all fields go to zero at $r = 0$.
+
+Converting the Poynting flux $P_{cyl}(r)$ in cylindrical coordinates for a point source at $r$ to 3d Cartesian coordinates involves multiplication by an $r$-dependent scale factor: $P_{3d} =  P_{cyl}(r) / r^2 \times 2 / \pi^5$. For a point source $r = 0$, the scale factor is $2 / (\pi \Delta r)^2$ where $\Delta r = \Delta z = 1/resolution$.
```

Comment:
This is confusing here.

When you do the Poynting flux integral etcetera, Meep automatically includes the 2πr factor.

However zero-thickness sources in the r direction are treated as δ(r) terms, where for some purposes you would rather have δ(r)/r or δ(r)/2πr, and if you want these factors you will need to include them manually (either in the source amplitude or by scaling the resulting computatations).